src/Shared/CliContext.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Tools.Internal
+{
+    public static class CliContext
+    {
+        /// <summary>
+        /// dotnet --verbose subcommand
+        /// </summary>
+        /// <returns></returns>
+        public static bool IsGlobalVerbose()
+        {
+            bool globalVerbose;
+            bool.TryParse(Environment.GetEnvironmentVariable("DOTNET_CLI_CONTEXT_VERBOSE"), out globalVerbose);
```

Comment:
A lot of environment variables work with `true` and `1`. Shouldn't you check that here? Or do the docs instruct people to only set this to `true` if they want to enable it?